Source/Fuse.PushNotifications/iOS/AppDelegatePushNotify.mm

```diff
@@ -9,17 +9,9 @@ - (void)application:(UIApplication *)application initializePushNotifications:(NS
 	if (launchOptions[UIApplicationLaunchOptionsRemoteNotificationKey]) {
 		[self application:application dispatchPushNotification:launchOptions[UIApplicationLaunchOptionsRemoteNotificationKey] fromBar:YES];
 	}
-	if ([application respondsToSelector:@selector(registerUserNotificationSettings:)]) {
-		// use registerUserNotificationSettings
-		[application registerUserNotificationSettings: [UIUserNotificationSettings settingsForTypes:(UIUserNotificationTypeSound  | UIUserNotificationTypeAlert | UIUserNotificationTypeBadge)  categories:nil]];
-		[application registerForRemoteNotifications];
-	} else {
-		// use registerForRemoteNotificationTypes:
-		[application registerForRemoteNotificationTypes:
-		 UIRemoteNotificationTypeBadge |
-		 UIRemoteNotificationTypeSound |
-		 UIRemoteNotificationTypeAlert];
-	}
+#if (!@(Project.iOS.PushNotifications.RegisterOnLaunch:IsSet)) || @(Project.iOS.PushNotifications.RegisterOnLaunch:Or(0))
```

Comment:
In fuselibs the only other place we use it with `#if` is Fuse.Platform/Android/SystemUI.uno#L117

A similar approach has been used to [splice in the the android senderID](https://github.com/fusetools/fuselibs-public/Source/Fuse.PushNotifications/Android/Impl.uno#L45) in push (which has been then for a year or two). In uxl files for local & push notifications and in WebView for the `Mobile.UriScheme`

In uno it's used in some [java code](Core/UnoCore/Targets/Android/@(Java.SourceDirectory)/com/fuse/App.java#L214) but mainly in build related uxl files.

It's not an ideal approach, I'd rather have the project data was included a queryable at runtime.